images/charmed-spark-gpu/conf/spark-defaults.conf

```diff
@@ -0,0 +1 @@
+spark.kubernetes.container.image=ghcr.io/canonical/charmed-spark:3.4-22.04
```

Comment:
Not sure whether this should be pointing to the new GPU image or the base spark image... (however until we have this released I think the integration tests won't pass because the gpu image is not published in GHCR yet..)